src/i18n/de-de/index.js

```diff
@@ -0,0 +1,363 @@
+        invalid_path_format: 'Der Pfad muss mit einem Schrägstrich enden /',
+        upload_file: 'Datei hochladen',
+        text_input: 'Texteingabe',
+        runs_value_for_optimization: 'Führt Wert zur Optimierung aus',
+        constructor_arguments: 'Constructor Arguments',
+        comma_seperated_values: 'Kommagetrennte Werte z.B.:, Bob,123,0x12345...',
+        no_trailing_commas: 'Keine nachgestellten Kommas',
+        paste_contract_code_here: 'Smart Contract Code kopieren und hier einfügen...',
+        enter_contract_text: 'Smart Contract Code eingeben oder einfügen',
+        verify_contract: 'Smart Contract verifizieren',
+        reset: 'Zurücksetzen',
+        gas_used: 'Verbrauchtes Gas',
+        transactions: 'Transaktionen',
+        account: 'Konto',
+        contract: 'Smart Contract',
+        erc20_transfers: 'ERC20 Transfers',
+        erc721_transfers: 'ERC721 Transfers',
+        erc1155_transfers: 'ERC1155 Transfers',
+        tokens: 'Tokens',
+        created_at_trx: 'Erstellt in Transaktion',
+        by_address: 'Durch Konto/Adresse',
+        number_used_once: 'Einmalig verwendete Nummer (nonce)',
+        native_account: 'Konto: Telos Native',
+        balance: 'Saldo',
+        view_source_prompt: 'Dieser Smart Contract wurde überprüft. Sie können den Quellcode und die Metadaten im Reiter \'contract\' ansehen',
+        account_url: '{ domain }/account/{ account }',
+        tlos_balance: '{ balance } TLOS',
+        couldnt_retreive_metadata_for_address: 'Metadata konnten nicht abgerufen werden { address }: { message }',
+        transaction_details: 'Transaktions-Details',
+        transaction_not_found: 'Nicht gefunden: { hash }',
+        general: 'Allgemein',
+        details: 'Details',
+        logs: 'Protokolle',
+        internal_txns: 'Interne Transaktionen',
+        transaction_hash: 'Transaktions Hash',
+        click_to_change_date_format: 'Klicken Sie hier, um das Datumsformat zu ändern',
+        block_number: 'Block Nummer',
+        from: 'Von',
+        to: 'Nach',
+        date: 'Datum',
+        success: 'Erfolg',
+        failure: 'Fehlgeschlagen',
+        status: 'Status',
+        balance_gwei: '{ amount } GWEI',
+        balance_tlos: '{ amount } TLOS',
+        error_message: 'Fehlermeldung',
+        contract_function: 'Smart Contract Funktionen',
+        function_parameters: 'Funktions-Parameter',
+        deployed_contract: 'Bereitgestellter Smart Contract',
+        click_to_show_in_wei: 'Anklicken zum Einblenden in wei',
+        gas_price_charged: 'Berechnete Transaktionskosten',
+        gas_fee: 'Transaktionskosten',
+        gas_limit: 'Transaktionskosten-Limit',
+        nonce: 'Nonce',
+        input: 'Eingabe',
+        output: 'Ausgabe',
+        value: 'Wert',
+    },
+    components: {
+        internal_txns: 'Interne Transaktionen',
+        n_internal_txns: '{ amount } interne transaktionen',
+        none: 'Keine',
```

Comment:
These 3 lines are new and need attention.

Reply:
done